src/Ilios/CoreBundle/Resources/config/routing.yml

```diff
@@ -96,6 +96,10 @@ api_curriculuminventoryexport_v1:
     prefix:   /v1
     defaults: {_format:json}
 
+api_curriculuminventorydownload_v1:
```

Comment:
Because this returns XML when nothing else does I think it may belong outside of the /api tree.  Possibly in /download or event its own /ciexport url space.